### ISSUE TITLE
Update 'data set' DOI to award DOI, multivalued, capture source/authority

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/submission-schema#224]
Currently, the Context page in the submission portal has "DOI associated with the data in this study. This is required when data is already generated."

What we're really asking for here is the award DOI from the user facility.

We need this if data was generated by JGI or EMSL. 
We can get it from the UP # that's provided, but would be easier if we could get people to add it themselves.

![Screenshot 2024-08-29 at 9 59 53 AM](https://github.com/user-attachments/assets/f890ecf8-a572-43ef-bf55-66b22915d9c8)

![Screenshot 2024-08-29 at 9 52 34 AM](https://github.com/user-attachments/assets/00827cfc-b960-4d51-865a-9edb8b93997c)

Example place to find award DOI: https://www.osti.gov/award-doi-service/biblio/10.46936/fics.proj.2017.49991/60006232
But, how does someone find that page?? 
@lamccue do you know?

**Comment by ssarrafan** [comment]
It's the last day of the sprint. I added @JamesTessmer to this ticket and will move it to the next sprint.  @mslarae13 @JamesTessmer let me know if this is done or if it's not being worked on right now.

**Comment by JamesTessmer** [comment]
This is in progress, I should have an associated PR up next week for some review.

**Comment by JamesTessmer** [comment]
@mslarae13 I've got a branch and PR up for this now. It should be set up how you described in the pictures above. I saw the issue mentions changing the field to be multivalued. Did we want to make a change for that similar to the funding source change?

**Comment by JamesTessmer** [comment]
@mslarae13 I've got a branch and PR up for this now. It should be set up how you described in the pictures above. I saw the issue mentions changing the field to be multivalued. Did we want to make a change for that similar to the funding source change?

**Comment by mslarae13** [comment]
Discussed today. Adding some clarity

Dataset DOI should be completely re-named to award DOI for step 2- Submission Context. Step 2 does NOT collect any DATA information

Any DOI provided on step 2 is an award_doi for DOI category https://microbiomedata.github.io/nmdc-schema/doi_category/

IF you select NO, data has not been generated.
- a award DOI box should appear under JGI & EMSL, if checked

If you select YES, data has been generated.
- the award DOI box should appear.
- you can add multiple, or leave this empty if you DO NOT select "doe user facility"
- IF the DOE user facility box is checked, award DOI is required

Who is the provider? 
- TBD for YES, leave it blank
- Use the user facility check box associated with the box that appear to fill in JGI or EMSL (or leave it blank & we can do this later)

**Comment by ssarrafan** [comment]
James is working on this now. Moving to next sprint.

**Comment by mslarae13** [comment]
Decision today at infrastructure sync: will not merge until after berkeley-schema is in production, so after oct 15. There will be merge conflicts

**Comment by ssarrafan** [comment]
Based on last comment I'll move this to the next sprint @JamesTessmer @mslarae13

**Comment by ssarrafan** [comment]
Berkeley was rolled out. @JamesTessmer can this be closed by tomorrow?

**Comment by JamesTessmer** [comment]
All the work for this is done and the PR is ready and passes checks. I think I’m just waiting for a final “go ahead” to hit the merge button.

**Comment by ssarrafan** [comment]
> All the work for this is done and the PR is ready and passes checks. I think I’m just waiting for a final “go ahead” to hit the merge button.

Final go ahead from @mslarae13?  I'll move to the next sprint since she's out this week.

**Comment by mslarae13** [comment]
Let's put this issue in next sprint, and maybe @pkalita-lbl and @JamesTessmer and I can sit down on a call and finalize it if needed or just get it merged in at the retreat

**Comment by JamesTessmer** [comment]
@mslarae13 I think Patrick tested it already and Mary approved the PR. I just tend to be overly cautious, but i think with your comment now its ok to merge in.

**Comment by mslarae13** [comment]
Going to test in dev
- did the data migrate?
- is the text correct

**Comment by ssarrafan** [comment]
Moving to next sprint for testing
@mslarae13 @JamesTessmer

**Comment by mslarae13** [comment]
@JamesTessmer  I was probably supposed to test this out earlier, and forgot! 

I just tested it, and it looks good. 
One confirmation, when I check JGI , an award box appears.. when I check EMSL, an award DOI box appears.
These aren't tied to the check UF, right? it's just there's probably 2 award DOIs so here's 2 boxes?

**Comment by JamesTessmer** [comment]
Yeah, i think the assumption was that if you are checking both you would have 2 DOIs
